### ISSUE TITLE
[Task prioritization] - Model training experiments

### ISSUE THREAD

**Issue by mdbfpz** [mdbfpz/krr_tool#50]
- Try different regularization methods: dropout, L2, ...
- Hyperparameter optimization
